tests/test_impls.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import os.path
+import tempfile
+from unittest.mock import Mock, patch
+
+import pytest
+
+from rapids_build_backend.impls import _edit_git_commit
+
+
+@pytest.mark.parametrize(
+    ["initial_contents"],
+    [
+        ("def456\n",),
+        ("",),
+        (None,),
+    ],
```

Comment:
Use the "easy" form of parametrize for single arguments. This might need reflowed.
```suggestion
    "initial_contents",
    [
        "def456\n",
        "",
        None,
    ],
```